### ISSUE TITLE
bug with team or Ressource manager

### ISSUE THREAD

**Issue by CPASimUSante** [claroline/Distribution#581]
On claroline 16-05, I've created a WS.
in that WS i've created some resources
then i went to create teams
First i create 2 teams (the button "create team**s**")
Then i create 1 team using the button "create team"

=> when i click back, on the WS resource manager (on the left side), i have : 
```
500 Internal Server Error - NonUniqueResultException
in vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php at line 781   -
        }
        if (count($result) > 1) {
            throw new NonUniqueResultException;
        }
        return array_shift($result);
at AbstractQuery ->getOneOrNullResult () 
in vendor/claroline/distribution/main/core/Repository/ResourceNodeRepository.php at line 70   + 
at ResourceNodeRepository ->findWorkspaceRoot (object(Workspace)) 
in vendor/claroline/distribution/main/core/Manager/ResourceManager.php at line 1280   + 
at ResourceManager ->getWorkspaceRoot (object(Workspace)) 
in vendor/claroline/distribution/main/core/Listener/Tool/ResourceManagerListener.php at line 162   + 
at ResourceManagerListener ->resourceWorkspace ('2') 
in vendor/claroline/distribution/main/core/Listener/Tool/ResourceManagerListener.php at line 103   + 
at ResourceManagerListener ->onDisplayWorkspaceResourceManager (object(DisplayToolEvent), 'open_tool_workspace_resource_manager', object(TraceableEventDispatcher)) 
at call_user_func (array(object(ResourceManagerListener), 'onDisplayWorkspaceResourceManager'), object(DisplayToolEvent), 'open_tool_workspace_resource_manager', object(TraceableEventDispatcher)) 
in vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php at line 61   + 
at WrappedListener ->__invoke (object(DisplayToolEvent), 'open_tool_workspace_resource_manager', object(ContainerAwareEventDispatcher)) 
at call_user_func (object(WrappedListener), object(DisplayToolEvent), 'open_tool_workspace_resource_manager', object(ContainerAwareEventDispatcher)) 
in app/cache/dev/classes.php at line 2427   + 
at EventDispatcher ->doDispatch (array(object(WrappedListener)), 'open_tool_workspace_resource_manager', object(DisplayToolEvent)) 
in app/cache/dev/classes.php at line 2356   + 
at EventDispatcher ->dispatch ('open_tool_workspace_resource_manager', object(DisplayToolEvent)) 
in vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php at line 124   + 
at TraceableEventDispatcher ->dispatch ('open_tool_workspace_resource_manager', object(DisplayToolEvent)) 
in vendor/claroline/distribution/main/core/Event/StrictDispatcher.php at line 78   + 
at StrictDispatcher ->dispatch ('open_tool_workspace_resource_manager', 'DisplayTool', array(object(Workspace))) 
in vendor/claroline/distribution/main/core/Controller/WorkspaceController.php at line 521   + 
at WorkspaceController ->openToolAction ('resource_manager', object(Workspace)) 
at call_user_func_array (array(object(WorkspaceController), 'openToolAction'), array('resource_manager', object(Workspace))) 
in app/bootstrap.php.cache at line 3109   + 
at HttpKernel ->handleRaw (object(Request), '1') 
in app/bootstrap.php.cache at line 3071   + 
at HttpKernel ->handle (object(Request), '1', true) 
in app/bootstrap.php.cache at line 3222   + 
at ContainerAwareHttpKernel ->handle (object(Request), '1', true) 
in app/bootstrap.php.cache at line 2444   + 
at Kernel ->handle (object(Request)) 
in web/app_dev.php at line 28   + 
```

Two other things to note : 
- i can access the ressource with the top resource manager (= non linked to a WS)
- the third team folder (created with the single team button), is created outside the WS folder : at root level 

=>
1. PersonnalWS 
2. TheWS -> with team 1 & 2 folder
3. team 3 folder

**Comment by LaurentGruber** [comment]
! si nom avec " ou caractère spécial ! voir que c'est traité correctement

**Comment by charlotte42** [comment]
Le problème n'a pas l'air de se produire quand l'accès au répertoire de l'équipe est privé.

**Comment by ngodfraind** [comment]
Reproduit.

**Comment by ngodfraind** [comment]
Fixed by #674
